modules/gltf/editor/editor_import_blend_runner.cpp

```diff
@@ -198,6 +198,40 @@ Error EditorImportBlendRunner::do_import(const Dictionary &p_options) {
 	}
 }
 
+HTTPClient::Status EditorImportBlendRunner::connect_blender_rpc(const Ref<HTTPClient> p_client, int p_timeout_usecs) {
```

Comment:
It should be with a `&`:
```suggestion
HTTPClient::Status EditorImportBlendRunner::connect_blender_rpc(const Ref<HTTPClient> &p_client, int p_timeout_usecs) {
```
Same in the header.